### PR TITLE
feat(lb): load balancer module

### PR DESCRIPTION
**Pull request by Monitob**

### CODE REVIEW

File: .terraform-docs.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    Terraform module that can be used to deploy Load Balancer resources on Scaleway. Common deployment examples can be found in [examples/](./examples).
+
+    ## Usage
+
+    The example below provision a basic Load Balancer.
+
+    ``` hcl
+    module "lb" {
+      source                          = "../../"
```
Comment (remyleone) [inline .terraform-docs.yml]: Is it normal to have a relative path here? What's that supposed to mean?

File: README.md [hunk 2]
```diff
@@ -1 +1,112 @@
...
+
+Terraform module that can be used to deploy Load Balancer resources on Scaleway. Common deployment examples can be found in [examples/](./examples).
+
+## Usage
+
+The example below provision a basic Load Balancer.
+
+``` hcl
+module "lb" {
+  source                          = "../../"
```
Comment (remyleone) [inline README.md]: Is this relative path rendered one way or another?

File: examples/basic/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Terraform module that can be used to deploy Load Balancer resources on Scaleway. Common deployment examples can be found in [examples/](./examples).
+
+This is a basic example showing how to use the module.
+
+## Usage
+
+The example below provision a basic Load Balancer.
+
+``` hcl
+    module "lb" {
```
Comment (remyleone) [inline examples/basic/README.md]: This block should be formatted with terraform fmt

File: examples/basic/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    source                          = "../../"
```
Comment (remyleone) [inline examples/basic/README.md]: I'm not sure to understand what this ../.. means

File: .terraform-docs.yml (hunk 1 above)
Comment (Monitob) [reply .terraform-docs.yml]: The pre-commit CI force you to put the source in this form, e.g on other [provider](https://github.com/terraform-aws-modules/terraform-aws-alb/blob/master/examples/complete-alb/main.tf#L32)

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+}
+
+variable "zone" {
+  description = "Load balancer zone"
+  type        = string
+  default     = "fr-par-1"
+}
+
+variable "create_lb" {
+  description = "Controls if the Load Balancer should be created"
```
Comment (remyleone) [inline variables.tf]: In which use-case with a load-balancer module, I'm not interested in creating a load-balancer? 🤔

Comment (Monitob) [reply variables.tf]: Variable removed

**Review by baalooos — APPROVED** [review]
LGTM

**Pull request merged**